<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Account::Internal::SitesWidget</class>
 <widget class="QWidget" name="Account::Internal::SitesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Working places</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="wpComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>100</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="minimumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="minimumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabAdress">
      <attribute name="title">
       <string>Adress</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>Name</string>
         </property>
         <property name="buddy">
          <cstring>nameEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="nameEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="adressLabel">
         <property name="text">
          <string>Adress</string>
         </property>
         <property name="buddy">
          <cstring>adressEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="adressEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="zipLabel">
         <property name="text">
          <string>ZIP</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="Utils::QButtonLineEdit" name="zip"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="cityLabel">
         <property name="text">
          <string>City</string>
         </property>
         <property name="buddy">
          <cstring>cityEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="Utils::QButtonLineEdit" name="cityEdit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="countryLabel">
         <property name="text">
          <string>Country</string>
         </property>
         <property name="buddy">
          <cstring>countryComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="Views::CountryComboBox" name="countryComboBox"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPhones">
      <attribute name="title">
       <string>Phones</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="phoneLabel">
         <property name="text">
          <string>Phone</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="buddy">
          <cstring>phoneEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="phoneEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="faxLabel">
         <property name="text">
          <string>Fax</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="buddy">
          <cstring>faxEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="faxEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="mailLabel">
         <property name="text">
          <string>Mail</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="buddy">
          <cstring>mailEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="mailEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabContacts">
      <attribute name="title">
       <string>Contacts</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="contactLabel">
         <property name="text">
          <string>Contact</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="contactEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Views::CountryComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">listviewplugin/countrycombobox.h</header>
  </customwidget>
  <customwidget>
   <class>Utils::QButtonLineEdit</class>
   <extends>QLineEdit</extends>
   <header>utils/widgets/qbuttonlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
